tensor_comprehensions/pybinds/tclib.cc

```diff
@@ -465,7 +482,13 @@ PYBIND11_MODULE(tclib, m) {
           "unchecked_run",
           &TcExecutor::uncheckedRun,
           py::arg("inputs"),
+          py::arg("outputs") = py::tuple())
+      .def(
+          "profile_kernel",
```

Comment:
Let's keep the names consistent with C++ API.  It should be called `profile`.

Reply:
profile returns a ProfileInfo structure that we don't want to expose, `profile_kernel` returns the kernel time in us.